include/safe/detail/concepts.hpp

```diff
@@ -15,4 +15,35 @@ concept range_like = requires(R r) {
     { r.begin() } -> iter_like;
     r.end();
 };
-} // namespace safe::detail
\ No newline at end of file
+
+template <char Char>
+constexpr bool is_decimal_digit_v = Char - '0' >= 0 && Char - '0' <= 9;
```

Comment:
**suggestion**: Why not `Char >= '0' and Char <= '9'` ? Is the subtraction/integral promotion giving us anything?

Reply:
Agree. No benefit from subtraction. Like to how the suggestion reads